pages/interactive.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable react/no-danger */
+
+import React from 'react';
+import Head from 'next/head';
+import styled from 'styled-components';
+import Layout from '../components/Layout';
+import Primary from '../components/Primary';
+import Card from '../components/Card';
+
+const title = 'Dynamic Data sets';
```

Comment:
I like "interactive" from the page name or "interactives" from Reuters. What do you think?

Reply:
I like either of those two, going to leave it as is for now and we can get a consensus at the meeting tomorrow.